test/CMakeLists.txt

```diff
@@ -906,3 +906,12 @@ if(MIOPEN_TEST_CONV)
     COMMAND	$<TARGET_FILE:test_conv2d>	--verbose	--input	1	48	7	7	--weights	1	48	5	5	--pads_strides_dilations	0	0	4	4	1	1
 )
 endif()
+
+# REGRESSION TEST for issue #894
+if(MIOPEN_BACKEND_OPENCL AND MIOPEN_TEST_ALL AND MIOPEN_TEST_HALF)
+    set(CONV_OCL_REGRESSION_TEST_ENV MIOPEN_FIND_MODE=normal MIOPEN_DEBUG_FIND_ONLY_SOLVER=ConvOclDirectFwd1x1)
```

Comment:
Please make it applicable for smoke, not only for MIOPEN_TEST_ALL. This test is small and fast enough for smoke.

Reply:
FYI, I've enabled it in smoke test.

Reply:
@ce1adon Thanks! Yes, I see that in another commit, but also I still see `SKIP_UNLESS_ALL` there ;)

Reply:
@shurale-nkn Let's add regression tests to "Full" stages for now. They will grow in the future.

Reply:
resolved